vernac/vernacprop.ml

```diff
@@ -0,0 +1,53 @@
+(************************************************************************)
```

Comment:
My point is that, not matter where vernac_classifies is or vernacprop is, they do very similar things: group vernac commands into classes.